com.unity.shadergraph/Editor/GraphUI/DataModel/Examples/ShaderGraphExampleTypes.cs

```diff
@@ -5,6 +5,7 @@
 using UnityEditor.ShaderGraph.Registry;
 using UnityEngine.GraphToolsFoundation.Overdrive;
 using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEditor.UIElements;
```

Comment:
Is this a duplicate import?

Reply:
Nope, this one is `UnityEditor` and the other is `UnityEngine`. The former has some editor-specific fields while the latter only contains the fundamentals.